pkg/middleware/request_tracing.go

```diff
@@ -57,6 +57,14 @@ var unnamedHandlers = []struct {
 
 // RouteOperationName receives the route operation name from context, if set.
 func RouteOperationName(req *http.Request) (string, bool) {
+	// all requests to the grafana k8s api server have a handler of /apis/*. This modifies that to be /apis/{kind}.
```

Comment:
```suggestion
	// all requests to the grafana k8s api server have a handler of /apis/*. This modifies that to be /apis/{group}.
```

Kind (or resource) is later in the UI